CMakeLists.txt

```diff
@@ -25,6 +25,8 @@
 # MATERIALS OR THE USE OR OTHER DEALINGS IN THE MATERIALS.
 
 cmake_minimum_required(VERSION 2.8.12)
+cmake_policy(SET CMP0054 NEW)
```

Comment:
This should be wrapped around  with`if(POLICY CMP0054)` since the policy was introduced in 3.1 and the minimum required cmake version for this file is 2.8.12.